R/methods.R

```diff
@@ -3484,6 +3491,39 @@ setGeneric("get_bibliography", function(.data)
 
 }
 
+#' get_bibliography
+#' @inheritParams get_bibliography
+#'
+#' @docType methods
+#' @rdname get_bibliography-methods
+#'
+#' @return A `tbl` with additional columns for the statistics from the hypothesis test (e.g.,  log fold change, p-value and false discovery rate).
```

Comment:
>@return A `tbl` with additional columns for the statistics from the hypothesis test (e.g.,  log fold change, p-value and false discovery rate).

Is that right? Or do you mean to say it's returning bibliography information? in BibTeX format?

If it needs changing there are also a few other cases of the same sentence below.